lib/_007/OpScope.pm6

```diff
@@ -39,6 +39,16 @@ class _007::OpScope {
         my $category = ~$0;
         my $op = ~$1;
 
+        if $category eq "infix" | "postfix" {
+            for <infix postfix> -> $other_category {
+                next if $category eq $other_category;
+                for %.ops{$other_category}.keys -> $other_op {
```

Comment:
Why is that for necessary?

Reply:
Improved with 9f11181a2352d3f3721d21d06dbfe99a7b00ed99.